i18n4go/i18n/init.go

```diff
@@ -49,22 +49,19 @@ var SUPPORTED_LOCALES = map[string]string{
 	"zh": "zh_CN",
 }
 var (
-	RESOUCES_PATH = filepath.Join("i18n", "resources")
-	bundle        *go_i18n.Bundle
+	RESOURCES_PATH = filepath.Join("i18n", "resources")
+	bundle         *go_i18n.Bundle
 )
 
 func GetResourcesPath() string {
-	return RESOUCES_PATH
+	return RESOURCES_PATH
 }
 
-func init() {
+func Init(packageName string, i18nDirname string, assetFn AssetFunc) TranslateFunc {
 	if bundle == nil {
 		bundle = go_i18n.NewBundle(language.AmericanEnglish)
 		bundle.RegisterUnmarshalFunc("json", json.Unmarshal)
 	}
-}
-
```

Comment:
**Callout**: Removed `init` here since it `bundle` is still nil during runtime. We need check if `bundle` is nil and initialize if it is each time we call the translate function `T(...)`